spec/acceptance/sqlserver2019rtm_2_instances_spec.rb

```diff
@@ -0,0 +1,38 @@
+# Encoding: utf-8
+require_relative 'spec_windowshelper'
+
+describe package('Microsoft SQL Server 2019 (64-bit)') do
+  it { should be_installed }
+end
+
+['SQL2019_1', 'SQL2019_2'].each do |instance_name|
+
+  describe service("MSSQL$#{instance_name}") do
+    it { should be_installed }
+    it { should be_enabled }
+    it { should be_running }
+    it { should have_start_mode('Automatic') }
+  end
+
+  describe windows_registry_key("HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Microsoft SQL Server\\MSSQL14.#{instance_name}\\Setup") do

```

Comment:
I suspect these will need bumping to 15 because that's the major version number of SQL 2019